spec/services/facebook_service_spec.rb

```diff
@@ -0,0 +1,68 @@
+require "rails_helper"
+
+# NOTE: Some of the requests being recorded by VCR here are quite long
+# so re-recording them might take a long time, in order of minutes per
+# request for some requests
```

Comment:
Because we're  creating a network of facebook test users and sadly, Koala does this one request at a time. This slowness is documented in the Koala wiki as well, to a degree.